src/io/iologindata.cpp

```diff
@@ -967,11 +969,6 @@ bool IOLoginData::savePlayer(Player* player)
   }
   query << " WHERE `id` = " << player->getGUID();
 
-  DBTransaction transaction;
```

Comment:
Removing this means that player level saves won't be consistent anymore.

Reply:
This removal was necassary as exaplained in #468, but i can reiterate on that. Let me make it clear with an example. Whenever one player has item in depot, and logs out, his depot state is commited to database(player 1 depot has the item). Then he logs back in, and takes item out, and hands it to second character. Now that second character puts the item into his depot and logs out and his depot state is commited(player 2 depot has the item). Note, that now both players have item in database, and both of those changes are commited. Now, is server crash happens, there will be two items in database, and item will become essentially duplicated. 

Also, after reading your comments, i see that my solution isn't ideal, and can still generate inconsistency on single player state. However i think that this is very unlikely, as it requires server to crash at the moment of server save. Never have i experienced a crash during server save on any open tibia server, but it might be just my observation.

Reply:
I'm centralising the discussion in the other thread, so we don't get lost. There I've explained why this won't be sufficient and why I believe that we shouldn't take this path.